packages/oui-stepper/_variables.less

```diff
@@ -0,0 +1,36 @@
+@oui-stepper-margin: 0 0 rem-calc(20);
+@oui-stepper-font-color: @oui-color-shuttle;
+@oui-stepper-font-size: rem-calc(16);
+@oui-stepper-font-weight: @oui-font-regular;
+
+@oui-stepper-step-padding: rem-calc(20);
+@oui-stepper-step-separator: 1px solid @oui-color-silver;
+
+@oui-stepper-step-container-padding-left: rem-calc(50);
+
+@oui-stepper-step-icon-background: @oui-color-sapphire;
+@oui-stepper-step-icon-background_complete: @oui-color-success-medium;
+@oui-stepper-step-icon-background_disabled: #e1e2e3;
+@oui-stepper-step-icon-color: #fff;
+@oui-stepper-step-icon-color_complete: #fff;
+@oui-stepper-step-icon-color_disabled: @oui-color-dusty;
+@oui-stepper-step-icon-font-size: rem-calc(20);
+@oui-stepper-step-icon-font-weight: @oui-font-semibold;
+@oui-stepper-step-icon-size: rem-calc(30);
+@oui-stepper-step-icon-top: rem-calc(5);
+
+@oui-stepper-step-title-line-height: rem-calc(22);
+@oui-stepper-step-title-color: @oui-color-sapphire;
+@oui-stepper-step-title-color_complete: @oui-color-sapphire;
+@oui-stepper-step-title-color_disabled: #b3b3b3;
```

Comment:
same "each color should have a name and be centralized for all component"

Reply:
It's use this way in others components. 
Maybe we should use `@oui-color-silver` instead (`#bbbdbf`)